src/main/java/com/pokegoapi/auth/GoogleLoginSecrets.java

```diff
@@ -0,0 +1,14 @@
+package com.pokegoapi.auth;
+
+/**
+ * Created by RajulB on 7/23/2016.
+ */
+
+public class GoogleLoginSecrets {
+	public static final String SECRET = "NCjF1TLi2CcY6t5mt0ZveuL7";
+	public static final String CLIENT_ID = "848232511240-73ri3t7plvk96pj4f85uj8otdat2alem.apps.googleusercontent.com";
+	public static final String OAUTH_ENDPOINT = "https://accounts.google.com/o/oauth2/device/code";
+	public static final String OAUTH_TOKEN_ENDPOINT = "https://www.googleapis.com/oauth2/v4/token";
+
+	public static String refresh_token = null;
```

Comment:
I have added a new login method which takes care of this. Ideally the old login method with just token_id as argument should be deprecated. Even if the user does not have a fresh token_id the new method would refresh it for him.